BEACON-V2-draft4-Model/genomicVariations/defaultSchema.json

```diff
@@ -242,7 +242,7 @@
       },
       "required": ["conditionId", "effect"]
     },
-    "FreqInPops": {
+    "FrequencyInPopulations": {
```

Comment:
Shouldn't this then be plural `FrequenciesInPopulations`? The it also wouldn't collide as much with the single `PopulationFrequency` entry label.

Reply:
Hmmm, there is just one frequency per population... and changing that name will break current implementations. Hence , if not a blocker, I suggest to stick with the singular option.

Reply:
Yes, that's why `PopulationFrequency` is correct; but you derive multiple frequencies if you have ...InPopulation**s**. (It is not completely wrong since you can read it as you mean; but ... strange; and it helps top make the distinction to the single entry `PopulationFrequency`).